understanding/20/pause-stop-hide.html

```diff
@@ -37,6 +37,11 @@ <h2>Intent of Pause, Stop, Hide</h2>
          </li>
          
       </ul>
+			
+      <p>Regarding the scope of condition (1) "starts automatically" as it relates to user interactions: for the animated information to start on a change of focus is considered automatic.  For the animated information to start on pointer hover, scrolling the page, or on changing of the setting of a user interface component is not considered automatic. 
```

Comment:
don't think the two 'of's are necessary.

```suggestion
      <p>Regarding the scope of condition (1) "starts automatically" as it relates to user interactions: for the animated information to start on a change of focus is considered automatic.  For the animated information to start on pointer hover, scrolling the page, or on changing the setting of a user interface component is not considered automatic. 
```